tests/integration_tests/security/test_spectre.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Tests ensuring SPECTREv2 mitigations are present."""
+
+import pytest
+
+from framework import utils
+from framework import defs
+
+
+@pytest.mark.timeout(100)
+def test_spectre_mitigations():
+    """
+    Run `spectre-mitigations` to check the kernel is compiled with SPECTREv2 mitigations.
+
+    @type: security
+    """
+    # Run command and raise exception if non-zero return code
+    utils.run_cmd("chmod a+x tests/integration_tests/security/spectre-mitigations", cwd=defs.FC_WORKSPACE_DIR)
```

Comment:
So this will run the chmod but not the script.

The script should be +x in the repo, and as part of this PR. To change the permissions on Windows https://stackoverflow.com/questions/21691202/how-to-create-file-execute-mode-permissions-in-git-on-windows

Reply:
Fixed as mentioned under  https://github.com/firecracker-microvm/firecracker/pull/3036#discussion_r900064268

Reply:
See https://github.com/firecracker-microvm/firecracker/pull/3036#discussion_r900152307